public/app/core/utils/richHistory.test.ts

```diff
@@ -10,7 +10,7 @@ import {
   filterAndSortQueries,
```

Comment:
Change of SortOrder import as we have moved LogsSortOrder to grafana/data and moved SortOrder for Rich history to app/core/utils/richHistory.